index.html

```diff
@@ -6,29 +6,32 @@
     <link type="text/css" rel="stylesheet" href="https://www.cyberbotics.com/jquery-ui/1.11.4/jquery-ui.min.css"/>
     <link type="text/css" rel="stylesheet" href="https://www.cyberbotics.com/highlight/9.2.0/default.min.css"/>
     <link type="text/css" rel="stylesheet" href="css/main.css"/>
+    <link type="text/css" rel="stylesheet" href="css/webots-doc.css"/>
   </head>
   <body>
-    <div id="left">
-      <div id="navigation">
-        <table>
-          <tr>
-            <td colspan="3"><a id="toc" title="Table of Contents">&equiv;</a></td>
-          </tr>
-          <tr>
-            <td><a id="previous" title="Previous page">&#x25C0;</a></td>
-            <td><a id="up" title="Up page">&#x25B2;</a></td>
-            <td><a id="next" title="Next page">&#x25B6;</a></td>
-          </tr>
-        </table>
+    <div class="webots-doc">
```

Comment:
Why adding a extra level of `div` here? That seems overkill...
Can't we use simply:
```html
<div class="webots-doc" id="left">
```